### ISSUE TITLE
deprecated warning in deps

### ISSUE THREAD

**Issue by Mister-Hope** [absolute-version/commit-and-tag-version#49]
```
stringify-package@1.0.1: This module is not used anymore, and has been replaced by @npmcli/package-json
```

I want to avoid deprecation warning from standard-version to this one, but your package is still showing some warnings, could you use the replaced one?

**Comment by TimothyJones** [comment]
Sure thing! I’ll take a look

**Comment by TimothyJones** [comment]
This is a little fiddly, as the new one is not a drop in replacement (the API for `@npmcli/package-json` is now async, whereas the previous one was synchronous - along with a couple of other changes).

I'll be able to set aside some time in early jan for this. If you'd like a fix sooner, PRs are very welcome - I can easily review and release.

**Comment by Merfoo** [comment]
Hi @TimothyJones, I've created a [PR](https://github.com/absolute-version/commit-and-tag-version/pull/65) to address this issue.
Let me know if you think the PR is a suitable solution to the issue.

**Comment by TimothyJones** [comment]
Love it! Thank you so much.

I added a comment. if you could sort out the license, then I'll get it merged and released asap.

**Comment by TimothyJones** [comment]
Released in 11.2.1, thanks again.
